kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -362,7 +362,7 @@ window.ToggleNavbarView = BaseView.extend ({
     initialize: function() {
 
         _.bindAll(this);
-        this.listenTo(this.model, "change", this.render);
+        this.listenTo(this.model, "change:client_server_time_diff", this.render);
```

Comment:
(Note: we don't just want something that only changes at page load, because the change in the status model after login is what should be triggering the change in the nav bar behaviour).